python/tank/hook.py

```diff
@@ -246,10 +247,22 @@ def get_publish_paths(self, sg_publish_data_list):
                                 "not contain a valid path definition" % sg_data)
 
             local_path = path_field.get("local_path")
-            if local_path is None:
+            url = path_field.get("url")
+            if local_path:
+                paths.append(local_path)
+
+            elif url and url.startswith("file://"):
```

Comment:
I am aware of the urlparse library, but in this case it felt overkill to me. I'll change it over.